src/core/requestService/UserPropertyRequests.ts

```diff
@@ -1,6 +1,10 @@
 import { logMethodCall } from "../../shared/utils/utils";
 import OneSignalError from "../../shared/errors/OneSignalError";
-import ExecutorResult from "../executors/ExecutorResult";
+import ExecutorResult, {
```

Comment:
unneeded import

Reply:
Import is need in this file, but just removed the 'default' import here.